packages/docs/tutorial/tutorial-requests.yaml

```diff
@@ -20,190 +20,213 @@ _ref:
     section: Tutorial
     filePath: tutorial/tutorial-requests.yaml
     prefetchPages:
```

Comment:
We don't support prefetch pages in Lowdefy V4 anymore, it is hanled by Next, so maybe we should just remove all of those configurations now

Reply:
Changes made in 9654eef9a0e0ece57d5e66f6400d8265e9346466